tutorials/quantum_simulation/Bayesian Optimisation for antiferromagnetic state preparation.ipynb

```diff
@@ -8,11 +8,6 @@
    ]
```

Comment:
<p>Somehow the results are noticeably worse now...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>The notebook now shows well, but this error remains. I does not seem to come from the simulator in itself, but I don't know yet where it is coming from.</p><p> </p>